docs/sources/setup-grafana/configure-security/configure-scim-provisioning/_index.md

```diff
@@ -0,0 +1,122 @@
+
+Grafana offers several methods for synchronizing users, teams, and roles. The table below compares SCIM with other synchronization methods to help you understand its advantages:
+
+| Sync Method                                                                    | Users | Teams | Roles | Automation | Key Benefits                                                             | Limitations                                                  | On-Prem | Cloud |
+| ------------------------------------------------------------------------------ | ----- | ----- | ----- | ---------- | ------------------------------------------------------------------------ | ------------------------------------------------------------ | ------- | ----- |
+| SCIM                                                                           | ✅    | ✅    | ⚠️    | Partial    | Complete user and team lifecycle management with automatic team creation | Requires SAML authentication; uses Role Sync for basic roles | ✅      | ✅    |
+| [Team Sync](../configure-team-sync/)                                           | ❌    | ✅    | ❌    | Partial    | Maps identity provider groups to Grafana teams                           | Requires manual team creation                                | ✅      | ✅    |
+| [Active LDAP Sync](../configure-authentication/enhanced-ldap/)                 | ✅    | ❌    | ❌    | Full       | Background synchronization of LDAP users                                 | Limited to LDAP environments                                 | ✅      | ❌    |
+| [Group Attribute Sync](../configure-group-attribute-sync/)                     | ❌    | ❌    | ✅    | Partial    | Maps identity provider group attributes to permissions                   | Limited to identity provider attributes                      | ✅      | ✅    |
+| [Role Sync](../configure-authentication/saml#configure-role-sync)              | ❌    | ❌    | ✅    | Partial    | Maps basic roles to users                                                | Limited to basic roles only                                  | ✅      | ✅    |
+| [Org Mapping](../configure-authentication/saml#configure-organization-mapping) | ❌    | ❌    | ✅    | Partial    | Maps basic roles per organization                                        | Only available for on-premises deployments                   | ✅      | ❌    |
+
+### Key advantages of SCIM over other methods
+
+- **Complete automation**: SCIM is the only method that fully automates user and team provisioning.
+- **Dynamic team creation**: Teams are created automatically based on identity provider groups, eliminating manual setup.
+- **Near real-time synchronization**: Changes in your identity provider are reflected in Grafana based on your identity provider synchronization schedule (consult your identity provider documentation for specific details).
+- **Enterprise-ready**: Designed for large organizations with complex user management needs.
```

Comment:
Consider making this list more parallel to help with readability: https://developers.google.com/tech-writing/one/lists-and-tables#keep_list_items_parallel